README.md

```diff
@@ -1,25 +1,31 @@
 Alerts v2
 =========
 
-Alerts for nib pages.
+Alerts for nib pages. There are two types of alerts - with and without a title.
 
-    <div class="v2-alert v2-alert--error">
-        <div class="v2-alert__title">Message Title</div>
-        Message content
+Alerts with a title:
+
+    <div class="alert alert--error">
+      <div class="alert__title">We're sorry, due to too many invalid login attempts your account is locked</div>
+      <div class="alert--message">You will be able to try again in 00:30 minutes.</div>
     </div>
 
-This component contains the following modifiers:
+Alerts without a title:
+    <div class="alert alert--error">
+      <div class="alert-msg--single">Alert message without title 18pt light Roboto</div>
```

Comment:
`title/message` would be an element rather than a modifier. You might do something like:
- Use an element modifier `alert__title alert__title--single` or `alert__message alert__message--single` (depending on whether you reckon its a title/message) OR
- Use a block modifier `alert alert--<type> alert--multiline` on the block and `alert__message` on the element